diff --git a/kernel_platform/build/kernel/kleaf/impl/stamp.bzl b/kernel_platform/build/kernel/kleaf/impl/stamp.bzl
--- a/kernel_platform/build/kernel/kleaf/impl/stamp.bzl
+++ b/kernel_platform/build/kernel/kleaf/impl/stamp.bzl
@@ -86,13 +86,12 @@
                 exit 1
             fi
             scmversion=""
-            oki_infix="o"
             stable_scmversion=$({stable_scmversion_cmd})
             scmversion_prefix=
             if [[ "$android_release" == "mainline" ]]; then
                 scmversion_prefix="-mainline"
             elif [[ -n "$android_release" ]] && [[ -n "$KMI_GENERATION" ]]; then
-                scmversion_prefix="-$android_release-$KMI_GENERATION-$oki_infix"
+                scmversion_prefix="-$android_release-$KMI_GENERATION"
             elif [[ -n "$android_release" ]] && [[ -z "$KMI_GENERATION" ]]; then
                 echo "KMI_GENERATION is not set but BRANCH is ${{BRANCH}}" >&2
                 exit 1
             elif [[ -z "$android_release" ]] && [[ -n "$KMI_GENERATION" ]]; then
